save-agent/src/linuxX64Main/kotlin/com/saveourtool/save/agent/AgentConfiguration.kt

```diff
@@ -68,12 +68,14 @@ data class HeartbeatConfig(
  * @property additionalDataEndpoint endpoint to post additional data (version etc.) to
  * @property executionDataEndpoint endpoint to post execution data to
  * @property filesEndpoint endpoint to post debug info to
+ * @property testSourceSnapshotEndpoint endpoint to download test source snapshots from
  */
 @Serializable
 data class BackendConfig(
     val url: String,
     val additionalDataEndpoint: String = "internal/saveAgentVersion",
     val executionDataEndpoint: String = "internal/saveTestResult",
+    val testSourceSnapshotEndpoint: String = "/internal/test-suites-sources/download-snapshot-by-execution-id",
```

Comment:
why the rest endpoints without `/` at the start

Reply:
I think having them starting with `/` is better, but didn't want to change existing paths in the same PR. Can be done as a part of #822